src/matrix/room/BaseRoom.js

```diff
@@ -559,6 +562,23 @@ export class BaseRoom extends EventEmitter {
         }
     }
 
+    async _getEventFromHomeserver(eventId) {
+        const response = await this._hsApi.context(this._roomId, eventId, 0).response();
+        const sender = response.event.sender;
+        const member = response.state.find(e => e.type === "m.room.member" && e.user_id === sender);
```

Comment:
you can import the "m.room.member" string from "./members/RoomMember.js" if you wish.